garage/tf/samplers/on_policy_vectorized_sampler.py

```diff
@@ -6,7 +6,7 @@
 from garage.misc import tensor_utils
 import garage.misc.logger as logger
 from garage.misc.overrides import overrides
-from garage.sampler import ProgBarCounter
```

Comment:
Can you put `ProgBarCounter` into a util somewhere in misc? It doesn't even belong in sampler.

Reply:
done